File: 2556-convert-the-temperature/convert-the-temperature.py
class Solution(object):
    def convertTemperature(self, celsius):
        """
        :type celsius: float
        :rtype: List[float]
        """
        Fahrenheit = celsius * 1.80 + 32.00
        Kelvin = celsius + 273.15

        return Kelvin,Fahrenheit

File: 2556-convert-the-temperature/convert-the-temperature.js
/**
 * @param {number} celsius
 * @return {number[]}
 */
var convertTemperature = function(celsius) {
    let Kelvin = celsius + 273.15
    let Fahrenheit = celsius * 1.80 + 32.00
    let number  = [Kelvin,Fahrenheit]

    return  number
};